assets/css/triptych_v2.scss

```diff
@@ -27,13 +27,26 @@
   line-height: 118px;
 }
 
+// Top level stuff
 @import "v2/triptych/screen_container";
+@import "v2/triptych/viewport";
 
+// Layout/slot elements
 @import "v2/triptych/screen/full_screen";
-@import "v2/train_crowding";
 
+// No-data states
+@import "v2/lcd_common_styles/page_load_no_data";
+@import "v2/lcd_common_styles/no_data";
+@import "v2/triptych/no_data";
```

Comment:
I'll try to get this reviewed / merged before the no-data branch. It's possible these may / may not be needed depending on the implementation of that, but we can leave that for the no-data pr to resolve.